metashare/api/jobs.py

```diff
@@ -195,6 +195,11 @@ def commit_changes_from_org(scratch_org, user, desired_changes, commit_message):
                 member_name_dict[member_type] = latest_revision_numbers[member_name][
                     member_type
                 ]
```

Comment:
This is building member_name_dict and then...doing nothing with it?

Reply:
We're mutating a dict that's on the `scratch_org` here and then saving it.